ferretdb/ferretdb.go

```diff
@@ -31,20 +31,18 @@ import (
 	"github.com/FerretDB/FerretDB/internal/clientconn"
 	"github.com/FerretDB/FerretDB/internal/clientconn/connmetrics"
 	"github.com/FerretDB/FerretDB/internal/handler/registry"
-	"github.com/FerretDB/FerretDB/internal/util/logging"
 	"github.com/FerretDB/FerretDB/internal/util/state"
 )
 
 // Config represents FerretDB configuration.
 type Config struct {
 	Listener ListenerConfig
 
-	// Deprecated: Use slog logger, panics if not nil.
+	// Setting this field triggers panic.
+	// Deprecated: Use Logger field instead.
 	Logger *zap.Logger
 
-	// slog logger to use; if nil, the default logger is used.
-	//
-	// To make logs accessible by `getLog` command, use [logging.WrapLogger] or [logging.WrapHandler].
```

Comment:
Use of this package can't import internal FerretDB packages, so those methods are unaccessible for them